persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/member/MemberDAO.kt

```diff
@@ -141,12 +158,20 @@ internal class MemberDAOImpl internal constructor(
     ) = withContext(coroutineContext) {
         memberQueries.transaction {
             userQueries.updateUserConnectionStatus(status, member.user)
-            val recordDidNotExist = userQueries.selectChanges().executeAsOne() == 0L
-            if (recordDidNotExist) {
+            val userRecordDidNotExist = userQueries.selectChanges().executeAsOne() == 0L
+            if (userRecordDidNotExist) {
                 userQueries.insertOrIgnoreUserIdWithConnectionStatus(member.user, status)
```

Comment:
cant we just insert or ignore form the beginning ?

Reply:
from how i understand this is we do update and check if it was not updated then we update or ignore will update or ignore will just do the same ?

Reply:
after talking to @Garzas we will do the change when merging with dev